Errors thrown when `yarn start` run

# 🐛 bug report

## Description of the problem
The following errors are thrown when setting up a local codesandbox development environment and running `yarn start`:

```
yarn run v1.6.0
$ yarn build:deps && lerna run start --scope app --stream
$ lerna run build:dev --scope codesandbox-{api,browserfs}
lerna info version 2.9.0
lerna info versioning independent
lerna info scope codesandbox-api
lerna ERR! build:dev Errored while running script in 'codesandbox-api'
lerna ERR! execute callback with error
lerna ERR! Error: Command failed: npm run build:dev codesandbox-browserfs
lerna ERR! fs.js:1657
lerna ERR!       binding.lstat(baseLong);
lerna ERR!               ^
lerna ERR! 
lerna ERR! Error: ENOENT: no such file or directory, lstat '/Users/nicknisi/code/codesandbox-client/packages/codesandbox-api/codesandbox-browserfs'
lerna ERR!     at Object.realpathSync (fs.js:1657:15)
lerna ERR!     at runRollup (/Users/nicknisi/code/codesandbox-client/node_modules/rollup-plugin-sourcemaps/node_modules/rollup/bin/rollup:1666:16)
lerna ERR!     at Object.<anonymous> (/Users/nicknisi/code/codesandbox-client/node_modules/rollup-plugin-sourcemaps/node_modules/rollup/bin/rollup:1777:2)
lerna ERR!     at Module._compile (module.js:652:30)
lerna ERR!     at Object.Module._extensions..js (module.js:663:10)
lerna ERR!     at Module.load (module.js:565:32)
lerna ERR!     at tryModuleLoad (module.js:505:12)
lerna ERR!     at Function.Module._load (module.js:497:3)
lerna ERR!     at Function.Module.runMain (module.js:693:10)
lerna ERR!     at startup (bootstrap_node.js:188:16)
lerna ERR! npm ERR! code ELIFECYCLE
lerna ERR! npm ERR! errno 1
lerna ERR! npm ERR! codesandbox-api@0.0.18 build:dev: `tsc && rollup -c "codesandbox-browserfs"`
lerna ERR! npm ERR! Exit status 1
lerna ERR! npm ERR! 
lerna ERR! npm ERR! Failed at the codesandbox-api@0.0.18 build:dev script.
lerna ERR! npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
lerna ERR! 
lerna ERR! npm ERR! A complete log of this run can be found in:
lerna ERR! npm ERR!     /Users/nicknisi/.npm/_logs/2018-05-08T21_37_28_570Z-debug.log
lerna ERR! 
lerna ERR! > codesandbox-api@0.0.18 build:dev /Users/nicknisi/code/codesandbox-client/packages/codesandbox-api
lerna ERR! > tsc && rollup -c "codesandbox-browserfs"
lerna ERR! 
lerna ERR! 
lerna ERR!     at Promise.all.then.arr (/Users/nicknisi/code/codesandbox-client/node_modules/execa/index.js:236:11)
lerna ERR!     at <anonymous>
{ Error: Command failed: npm run build:dev codesandbox-browserfs
fs.js:1657
      binding.lstat(baseLong);
              ^

Error: ENOENT: no such file or directory, lstat '/Users/nicknisi/code/codesandbox-client/packages/codesandbox-api/codesandbox-browserfs'
    at Object.realpathSync (fs.js:1657:15)
    at runRollup (/Users/nicknisi/code/codesandbox-client/node_modules/rollup-plugin-sourcemaps/node_modules/rollup/bin/rollup:1666:16)
    at Object.<anonymous> (/Users/nicknisi/code/codesandbox-client/node_modules/rollup-plugin-sourcemaps/node_modules/rollup/bin/rollup:1777:2)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! codesandbox-api@0.0.18 build:dev: `tsc && rollup -c "codesandbox-browserfs"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the codesandbox-api@0.0.18 build:dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nicknisi/.npm/_logs/2018-05-08T21_37_28_570Z-debug.log

> codesandbox-api@0.0.18 build:dev /Users/nicknisi/code/codesandbox-client/packages/codesandbox-api
> tsc && rollup -c "codesandbox-browserfs"


    at Promise.all.then.arr (/Users/nicknisi/code/codesandbox-client/node_modules/execa/index.js:236:11)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
  code: 1,
  killed: false,
  stdout: '\n> codesandbox-api@0.0.18 build:dev /Users/nicknisi/code/codesandbox-client/packages/codesandbox-api\n> tsc && rollup -c "codesandbox-browserfs"\n\n',
  stderr: 'fs.js:1657\n      binding.lstat(baseLong);\n              ^\n\nError: ENOENT: no such file or directory, lstat \'/Users/nicknisi/code/codesandbox-client/packages/codesandbox-api/codesandbox-browserfs\'\n    at Object.realpathSync (fs.js:1657:15)\n    at runRollup (/Users/nicknisi/code/codesandbox-client/node_modules/rollup-plugin-sourcemaps/node_modules/rollup/bin/rollup:1666:16)\n    at Object.<anonymous> (/Users/nicknisi/code/codesandbox-client/node_modules/rollup-plugin-sourcemaps/node_modules/rollup/bin/rollup:1777:2)\n    at Module._compile (module.js:652:30)\n    at Object.Module._extensions..js (module.js:663:10)\n    at Module.load (module.js:565:32)\n    at tryModuleLoad (module.js:505:12)\n    at Function.Module._load (module.js:497:3)\n    at Function.Module.runMain (module.js:693:10)\n    at startup (bootstrap_node.js:188:16)\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! codesandbox-api@0.0.18 build:dev: `tsc && rollup -c "codesandbox-browserfs"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the codesandbox-api@0.0.18 build:dev script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /Users/nicknisi/.npm/_logs/2018-05-08T21_37_28_570Z-debug.log\n',
  failed: true,
  signal: null,
  cmd: 'npm run build:dev codesandbox-browserfs',
  timedOut: false,
  exitCode: 1 }
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## How has this issue affected you? What are you trying to accomplish?
It is unclear how to build codesandbox locally with these errors being thrown. @lbogdan recommended the following commands in Discord, which does work around the issue and allow for local development with the `yarn start:fast`.

```
(cd packages/codesandbox-api && yarn build:dev)
(cd standalone-packages/codesandbox-browserfs && yarn build:dev)
yarn start:fast
```

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  | not applicable |
| Browser | not applicable |
| Operating System | macOS 10.13.4 |

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "yarn build:deps && yarn build:prod",
     "build:prod": "lerna run build --scope app --scope homepage --parallel && gulp",
-    "build:deps": "lerna run build:dev --scope codesandbox-{api,browserfs}",
+    "build:deps": "lerna run build:dev --scope codesandbox-api --scope codesandbox-browserfs",
     "start": "yarn build:deps && lerna run start --scope app --stream",
     "start:fast": "lerna run start --scope app --stream",
     "start:home": "lerna run start --scope homepage --stream",

